pages/forgot-password.js

```diff
@@ -0,0 +1,108 @@
+import API_URL from "../components/globalApiUrl.js";
```

Comment:
separate the imports by the one from libraries at the top and the ones from local components after, separated by a space.